diff --git a/Make/options b/Make/options
index 79305a0..627b684 100644
--- a/Make/options
+++ b/Make/options
@@ -17,8 +17,9 @@ EXE_INC = \
 -I$(LIB_SRC)/OpenFOAM/lnInclude \
 -I$(LIB_SRC)/meshTools/lnInclude \
 -I$(LIB_SRC)/sampling/lnInclude \
-$(INCOMPRESSIBLE_TURB_INC) \
-$(INCOMPRESSIBLE_TRANSPORT_INC) \
+-I$(LIB_SRC)/TurbulenceModels/incompressible/lnInclude \
+-I$(LIB_SRC)/TurbulenceModels/turbulenceModels/lnInclude \
+-I$(LIB_SRC)/transportModels/incompressible/lnInclude \
 -IversionRules \
 -IrootFinding/RootFinder \
 -IlawsOfTheWall/LawOfTheWall \
@@ -34,5 +35,5 @@ LIB_LIBS = \
 -lOpenFOAM \
 -lmeshTools \
 -lsampling \
-$(INCOMPRESSIBLE_TURB_LIB) \
-$(INCOMPRESSIBLE_TURB_ALL_LIBS)
+-lincompressibleTurbulenceModels \
+-lturbulenceModels
diff --git a/samplers/MultiCellSampler/MultiCellSampler.C b/samplers/MultiCellSampler/MultiCellSampler.C
index 985dda7..75be8fc 100644
--- a/samplers/MultiCellSampler/MultiCellSampler.C
+++ b/samplers/MultiCellSampler/MultiCellSampler.C
@@ -22,6 +22,7 @@ License
 #include "meshSearch.H"
 #include "fvPatchFieldMapper.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 #include "wallFvPatch.H"
 #include "objectRegistry.H"
 #include "IOField.H"
diff --git a/samplers/SampledField/SampledPGradField.C b/samplers/SampledField/SampledPGradField.C
index 71dcae8..218dcde 100644
--- a/samplers/SampledField/SampledPGradField.C
+++ b/samplers/SampledField/SampledPGradField.C
@@ -20,6 +20,7 @@ License
 
 #include "SampledPGradField.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 #include "fvcGrad.H"
 #include "List.H"
 #include "helpers.H"
diff --git a/samplers/SampledField/SampledVelocityField.C b/samplers/SampledField/SampledVelocityField.C
index 0bde999..77d94bd 100644
--- a/samplers/SampledField/SampledVelocityField.C
+++ b/samplers/SampledField/SampledVelocityField.C
@@ -20,6 +20,7 @@ License
 
 #include "SampledVelocityField.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 #include "helpers.H"
 #include "interpolation.H"
 //
diff --git a/samplers/SampledField/SampledWallGradUField.C b/samplers/SampledField/SampledWallGradUField.C
index 8fbab90..9079931 100644
--- a/samplers/SampledField/SampledWallGradUField.C
+++ b/samplers/SampledField/SampledWallGradUField.C
@@ -20,6 +20,7 @@ License
 
 #include "SampledWallGradUField.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 #include "codeRules.H"
 #include "helpers.H"
 //
diff --git a/samplers/SingleCellSampler/SingleCellSampler.C b/samplers/SingleCellSampler/SingleCellSampler.C
index 9ca0bfc..326b2bf 100644
--- a/samplers/SingleCellSampler/SingleCellSampler.C
+++ b/samplers/SingleCellSampler/SingleCellSampler.C
@@ -22,6 +22,7 @@ License
 #include "meshSearch.H"
 #include "fvPatchFieldMapper.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 #include "wallFvPatch.H"
 #include "objectRegistry.H"
 #include "IOField.H"
diff --git a/scalarListListIOList/scalarListListIOList.C b/scalarListListIOList/scalarListListIOList.C
index 7ac9272..85d61bf 100644
--- a/scalarListListIOList/scalarListListIOList.C
+++ b/scalarListListIOList/scalarListListIOList.C
@@ -18,6 +18,7 @@ License
 
 \*---------------------------------------------------------------------------*/
 
+#define Foam_IOList_defines_typeName
 #include "scalarListListIOList.H"
 #include "addToRunTimeSelectionTable.H"
 
diff --git a/wallModels/wallModelFvPatchScalarField.H b/wallModels/wallModelFvPatchScalarField.H
index 6717546..1f6c241 100644
--- a/wallModels/wallModelFvPatchScalarField.H
+++ b/wallModels/wallModelFvPatchScalarField.H
@@ -50,6 +50,7 @@ SourceFiles
 
 #include "fixedValueFvPatchFields.H"
 #include "volFields.H"
+#include "surfaceFields.H"
 
 // * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
 
